README.md

```diff
@@ -1705,6 +1705,8 @@ The following extractors use this feature:
 * `comment_sort`: `top` or `new` (default) - choose comment sorting mode (on YouTube's side)
 * `max_comments`: Limit the amount of comments to gather. Comma-separated list of integers representing `max-comments,max-parents,max-replies,max-replies-per-thread`. Default is `all,all,all,all`
     * E.g. `all,all,1000,10` will get a maximum of 1000 replies total, with up to 10 replies per thread. `1000,all,100` will get a maximum of 1000 comments, with a maximum of 100 replies total
+* `innertube_host`: Override Innertube API host to use for all API requests (e.g. `youtubei.googleapis.com`)
+* `innertube_key`: Override Innertube API key to use for all API requests
```

Comment:
We only discussed `innertube_host` on discord, no? idk where custom keys are useful. But assuming they are, the PR can be merged at your discretion

Reply:
some keys are restricted to the host/origin

e.g. the key we use for web_music can only be used with music.youtube.com. 

so if the user wants to override the host for that client as well they may need to configure a different key